### PR TITLE
README: link to Winget manifest, add full package identifier install option

### PR DESCRIPTION
**Pull request by kbdharun**

### CODE REVIEW

**Comment by flxzt** [comment]
Thanks. What's the reason for adding "FelixZwettler.Rnote" as additional installation command?

**Comment by kbdharun** [comment]
> What's the reason for adding "FelixZwettler.Rnote" as additional installation command?

Actually, this is the primary intended form for specifying packages in Winget commands as it won't lead to package name conflicts. Currently, the rnote package term (moniker) is only referenced by us so the current command is fine. I though of adding the other one to stay consistent with other install command for FlatHub, etc.

**Comment by flxzt** [comment]
Sounds good, we could also remove the possible ambigious command and keep the one with the full identifier. Is there a possibility to change the prefix to `flxzt` as well similar to the app id that is used throughout the app and flathub? I'm not completely comfortable with my regular full name being used there.

**Comment by kbdharun** [comment]
> Is there a possibility to change the prefix to flxzt as well similar to the app id that is used throughout the app and flathub? I'm not completely comfortable with my regular full name being used there.

Sure, it is possible to do it. Although not straightforward (as one manifest can be modified in a PR). I need to remove the current manifests and it's directory, and create the renamed manifest in a group of PRs which I will do now.

File: README.md [hunk 1]
```diff
@@ -76,10 +76,11 @@ Download the Windows installer from the latest release which can be found [here]
 <a href='https://github.com/flxzt/rnote/releases/latest'><img width="250" alt='Download Windows installer' src='https://raw.githubusercontent.com/flxzt/rnote/main/misc/assets/windows-installer-badge.svg'/></a>
 </div><br>
 
-Install using winget:
+Install using [Winget](https://github.com/microsoft/winget-pkgs/tree/master/manifests/f/FelixZwettler/Rnote):
 
 ```
 winget install rnote
+winget install FelixZwettler.Rnote
```
Comment (kbdharun) [inline README.md:83]:
```suggestion
winget install flxzt.Rnote
```

File: README.md [hunk 2]
```diff
@@ -76,10 +76,11 @@ Download the Windows installer from the latest release which can be found [here]
 <a href='https://github.com/flxzt/rnote/releases/latest'><img width="250" alt='Download Windows installer' src='https://raw.githubusercontent.com/flxzt/rnote/main/misc/assets/windows-installer-badge.svg'/></a>
 </div><br>
 
-Install using winget:
+Install using [Winget](https://github.com/microsoft/winget-pkgs/tree/master/manifests/f/FelixZwettler/Rnote):
 
 ```
 winget install rnote
+winget install flxzt.Rnote
```
Comment (kbdharun) [inline README.md:83]:
```suggestion
winget install flxzt.rnote
```

File: README.md [hunk 3]
```diff
@@ -76,10 +76,11 @@ Download the Windows installer from the latest release which can be found [here]
 <a href='https://github.com/flxzt/rnote/releases/latest'><img width="250" alt='Download Windows installer' src='https://raw.githubusercontent.com/flxzt/rnote/main/misc/assets/windows-installer-badge.svg'/></a>
 </div><br>
 
-Install using winget:
+Install using [Winget](https://github.com/microsoft/winget-pkgs/tree/master/manifests/f/FelixZwettler/Rnote):
 
 ```
 winget install rnote
```
Comment (kbdharun) [inline README.md:82]:
```suggestion
```

**Comment by flxzt** [comment]
the link would need to be update to: https://github.com/microsoft/winget-pkgs/tree/master/manifests/f/flxzt/rnote

File: README.md [hunk 4]
```diff
@@ -76,10 +76,10 @@ Download the Windows installer from the latest release which can be found [here]
 <a href='https://github.com/flxzt/rnote/releases/latest'><img width="250" alt='Download Windows installer' src='https://raw.githubusercontent.com/flxzt/rnote/main/misc/assets/windows-installer-badge.svg'/></a>
 </div><br>
 
-Install using winget:
+Install using [Winget](https://github.com/microsoft/winget-pkgs/tree/master/manifests/f/FelixZwettler/Rnote):
```
Comment (kbdharun) [inline README.md:79]:
```suggestion
Install using [Winget](https://github.com/microsoft/winget-pkgs/tree/master/manifests/f/flxzt/rnote):
```

**Pull request merged**